quinn-proto/src/crypto.rs

```diff
@@ -30,7 +30,7 @@ pub trait Session: Sized {
     /// Type used to represent packet protection keys
     type Keys: Keys + Sized;
     /// Type used to hold configuration for server sessions
-    type ServerConfig: ServerConfig<Self>;
+    type ServerConfig: ServerConfig<Self> + Default;
```

Comment:
The `Default` bound should be placed on `impl Default for ServerConfig` where it's specifically needed, rather than made mandatory.

Reply:
Right -- this pretty much keeps it more similar to what it was before. Done.